app/index.js

```diff
@@ -1,27 +1,26 @@
-require('./insights').setup()
-require('log-timestamp')
-const messaging = require('./messaging')
-const notifyMonitor = require('./email/notify-monitor')
-const createServer = require('./server')
+import { setup } from './insights'
+import 'log-timestamp' // Unsure what this does?
+import { startMessaging, stopMessaging } from './messaging'
+import { start as startNotifyMonitor } from './email/notify-monitor'
+import { createServer } from './server'
 
 const init = async () => {
+  await startMessaging()
+  await startNotifyMonitor()
   const server = await createServer()
   await server.start()
-  console.log('Server running on %s', server.info.uri)
+  console.log(`Server running on ${server.info.uri}`)
+  setup()
 }
 
 process.on('SIGTERM', async () => {
-  await messaging.stop()
+  await stopMessaging()
   process.exit(0)
 })
 
 process.on('SIGINT', async () => {
-  await messaging.stop()
+  await stopMessaging()
   process.exit(0)
 })
 
-module.exports = (async function startService () {
-  await messaging.start()
-  await notifyMonitor.start()
-  await init()
-}())
+init()
```

Comment:
Does this work? I found with one of the other apps I had to await this async function here, or the hapi server wouldn't start when I ran the app

Reply:
Yeah it works, I dont think it makes any difference if we await it or not, and if we want to await it we need to make config changes